modules/thermo-cycler/thermo-cycler-arduino/lid.cpp

```diff
@@ -266,9 +270,17 @@ bool Lid::close_cover()
   }
   motor_on();
   bool res = move_angle(-LID_MOTOR_RANGE_DEG);
-  delay(500);
   if (res)
   {
+#if HW_VERSION >=4
+    // move down a bit
```

Comment:
Same thing about why rather than what - is this to make sure we hit the optical switch?